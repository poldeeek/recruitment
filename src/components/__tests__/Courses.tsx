import Courses from '../Courses';
import { render } from '@testing-library/react';

jest.mock('../../utils/fetcher', () => {
  return {
    fetcher: jest.fn().mockImplementation(() => {
      return 'Some data';
    }),
    readData: jest.fn(),
  };
});

jest.mock('../CollapsibleTable', () => {
  const CollapsibleTable = () => <div>CollapsibleTable</div>;
  return CollapsibleTable;
});

test('Basic redner', () => {
  const component = render(<Courses />);
  expect(component.baseElement).toMatchSnapshot();
});
