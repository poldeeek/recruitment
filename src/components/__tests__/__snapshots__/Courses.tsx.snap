// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic redner 1`] = `
<body>
  <div>
    <div
      class="sc-dlnjwi fNuxOi"
    />
  </div>
</body>
`;
